server_extensions_extra/src/main/resources/batm-extensions.xml

```diff
@@ -61,14 +61,19 @@
                 <cryptocurrency>BTC</cryptocurrency>
                 <cryptocurrency>ETH</cryptocurrency>
             </ratesource>
-            <ratesource prefix="mrcoin" name ="MrCoin.eu" >
-                <cryptocurrency>BTC</cryptocurrency>
-                <cryptocurrency>ETH</cryptocurrency>
-                <cryptocurrency>BCH</cryptocurrency>
-                <cryptocurrency>LTC</cryptocurrency>
-                <cryptocurrency>DASH</cryptocurrency>
-                <cryptocurrency>DAI</cryptocurrency>
-            </ratesource>
+        <ratesource prefix="mrcoin" name ="MrCoin.eu" >
+            <cryptocurrency>BTC</cryptocurrency>
+            <cryptocurrency>ETH</cryptocurrency>
+            <cryptocurrency>BCH</cryptocurrency>
+            <cryptocurrency>LTC</cryptocurrency>
+            <cryptocurrency>DASH</cryptocurrency>
+            <cryptocurrency>DAI</cryptocurrency>
+        </ratesource>
+        <ratesource prefix="bitkub" name ="BitKub.com" >
+            <param name="fiatcurrency" />
+            <cryptocurrency>BTC</cryptocurrency>
```

Comment:
Add here also ETH, BCH, LTC as you defined them in the rate source.

Reply:
Done. ;-)

                <cryptocurrency>BTC</cryptocurrency>
                <cryptocurrency>ETH</cryptocurrency>
                <cryptocurrency>BCH</cryptocurrency>
                <cryptocurrency>LTC</cryptocurrency>
                <cryptocurrency>DOGE</cryptocurrency>
                <cryptocurrency>USDT</cryptocurrency>
                <cryptocurrency>XRP</cryptocurrency>

and:


        result.add(CryptoCurrency.BTC.getCode());
        result.add(CryptoCurrency.ETH.getCode());
        result.add(CryptoCurrency.BCH.getCode());
        result.add(CryptoCurrency.LTC.getCode());
        result.add(CryptoCurrency.DOGE.getCode());
        result.add(CryptoCurrency.USDT.getCode());
        result.add(CryptoCurrency.XRP.getCode());